qml/keys/SpaceKey.qml

```diff
@@ -56,6 +56,9 @@ ActionKey {
         }
 
         onPressed: {
+            if (maliit_input_method.useHapticFeedback) {
+                pressEffect.start();
+               }
```

Comment:
Please fix the alignment of the closing brace. It should be aligned with the `i` in the `if` statement (4 spaces indented from the parent `onPressed: {` line).